auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -949,4 +767,20 @@ export default defineComponent({
   border-style: solid;
   border-color: transparent transparent rgba(50, 50, 50, 0.7) transparent;
 }
+
+.status-dot {
+  height: 8px;
+  width: 8px;
+  border-radius: 50%;
+  display: inline-block;
+  margin-right: 8px;
+}
+
+.status-dot.AWAITING_REVIEW { background-color:#F8661A; }
+.status-dot.CHANGE_REQUESTED { background-color:#2c56c7; }
+.status-dot.RESUBMITTED { background-color:#D3272C; }
+.status-dot.REJECTED { background-color: lightgray; }
+.status-dot.APPROVED { background-color: lightgray; }
+.status-dot.ABANDONED { background-color: lightgray; }
+.status-dot.CANCELLED { background-color: lightgray; }
```

Comment:
I like this layout (easy to understand).

My only comment is that I'd like to see named instead of hex colours. You can use colour variables here. See [theme.scss](https://github.com/bcgov/sbc-auth/blob/55062f471e92ec6301284729c34127c34d3af7ec/auth-web/src/assets/scss/theme.scss#L83).

Reply:
Up to you but you could also nest the statuses under the main classname, like this:
```
.status-dot {
   blah blah blah;

   &.AWAITING_REVIEW { blah blah blah; }
   &...
}
```

Reply:
Thanks, will update

Reply:
Updated